src/OrchardCore.Themes/TheBlogTheme/Recipes/blog.recipe.json

```diff
@@ -51,10 +51,13 @@
         "OrchardCore.ContentFields",
         "OrchardCore.ContentPreview",
         "OrchardCore.Contents",
+        "OrchardCore Deployment",
+        "OrchardCore.Deployment.Remote",
+        "OrchardCore.Contents.Deployment.AddToDeploymentPlan",
+        "OrchardCore.Contents.Deployment.Download",
+        "OrchardCore.Contents.Deployment.ExportContentToDeploymentTarget",
```

Comment:
After changing the blog recipe to this, using the blog recipe when adding tenants will reproduce the problem
@gvkries

Reply:
Ok, thanks. I think I understand the problem. Give me a sec...